### PR TITLE
Add the moon's track on the sky to the observing example (not ready yet)

### PR DESCRIPTION
**Pull request by eteq**
This is a simple PR that just adds a line for the track of the moon to the observing example.

The reason I say it's not ready to merge yet is that, as it stands, this requires both a 10MB download and that `jplephem` be installed to build the docs.  We probably don't want to make people deal with that.  But #4963 is in, there'll be a "builtin" (i.e. approximation that requires no download) version of `get_moon`, and at that time this should be changed to use that and merged.

### CODE REVIEW

File: examples/coordinates/plot_obs-planning.py [hunk 1]
```diff
@@ -117,6 +117,16 @@
...
 sunaltazs_July12_to_13 = get_sun(times_July12_to_13).transform_to(frame_July12_to_13)
 
+
+##############################################################################
+# Do the same with `~astropy.coordinates.get_moon` to find when the moon is
+# up. Be aware that this will need to download a 10MB file from the internet
+# to get a precise location of the moon.
+
+from astropy.coordinates import get_moon
+moon_July12_to_13 = get_moon(times_July12_to_13, ephemeris='de432s')
```
Comment (adrn) [inline examples/coordinates/plot_obs-planning.py]: Remove the `ephemeris` argument?

**Comment by adrn** [comment]
Just a friendly note that #4963 is merged!

**Comment by bsipocz** [comment]
@eteq - could you click the box for "allow edits from maintainers" here? I've rebased, checked the rendered version this and addressed @adrn's comment above, but cannot push it back to your branch.

**Comment by eteq** [comment]
Done!

**Comment by pllim** [comment]
I got a tiny heart attack when I got a notification that a PR title "not ready yet" got merged. But I trust you, @bsipocz !

**Comment by bsipocz** [comment]
Oh, I haven't noticed that, sorry. But I built the doc and compared to the
current version and it seemed reasonable,  and based on the comments it
seemed to be a finished PR.

**Comment by eteq** [comment]
There, I fixed it @pllim :wink: it was ready after @bsipocz's commit 532ca48, which addressed @adrn's suggestion.

**Pull request merged**